test/hotel_block_test.rb

```diff
@@ -0,0 +1,108 @@
+require 'test_helper'
```

Comment:
I needed to change this to match your other tests before I could run rake. 
```suggestion
require_relative 'test_helper'
```